app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -63,11 +67,46 @@ export const ProjectBoard = ({
             state: status.UNSTARTED
           })}
         />,
-      children: <Sprints sprints={backlogSprints} />
+      children: <Sprints sprints={backlogSprints} />,
+      visible: projectBoard.visibleColumns.backlog,
+      onClose: () => toggleColumn('backlog')
     },
     {
       title: I18n.t("projects.show.done"),
-      children: <Sprints sprints={doneSprints} fetchStories={fetchPastStories} />
+      children: <Sprints sprints={doneSprints} fetchStories={fetchPastStories} />,
+      visible: projectBoard.visibleColumns.done,
+      onClose: () => toggleColumn('done')
+    }
+  ];
+
+  const sideBarButtons = [
+    {
+      description: I18n.t('revert_columns_tooltip'),
+      onClick: reverseColumns,
+      'data-id': 'reverse-button',
+      toggled: projectBoard.reverse,
+      icon: 'fas fa-columns'
+    },
+    {
+      description: I18n.t('toggle_column', { column: 'chilly bin' }),
+      onClick: () => toggleColumn('chillyBin'),
+      'data-id': 'toggle-chilly-bin',
+      toggled: projectBoard.visibleColumns.chillyBin,
```

Comment:
I think the word `toggled` here may be a little confusing, because "toggling" means "inverting the state". What do you think about replacing it with `isVisible`?